src/components/KeyboardLayout/KeyboardSelect.tsx

```diff
@@ -0,0 +1,14 @@
+import Select from "react-select";
+import { KBLayout } from "../../enums/KBLayout";
+
+const keys = Object.keys(KBLayout);
```

Comment:
Can you move this inside the component?
We don't generally want variables outside of the component